inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/scatter_nd_update.cpp

```diff
@@ -16,8 +16,9 @@ namespace {
 // map<inputShape map<indicesShape, indicesValue>>
 // updateShape is gotten from inputShape and indicesShape
 std::map<std::vector<size_t>, std::map<std::vector<size_t>, std::vector<size_t>>> sliceSelectInShape {
+    {{4, 3, 2, 3, 2}, {{{2, 2, 1}, {3, 2, 0, 1}}}},
     {{10, 9, 9, 11}, {{{4, 1}, {1, 3, 5, 7}}, {{1, 2}, {4, 6}}, {{2, 3}, {0, 1, 1, 2, 2, 2}}, {{1, 4}, {5, 5, 4, 9}}}},
-    {{10, 9, 10, 9, 10}, {{{2, 3}, {0, 4, 6, 5, 7, 1}}}},
+    {{10, 9, 10, 9, 10}, {{{2, 2, 1}, {5, 6, 2, 8}}, {{2, 3}, {0, 4, 6, 5, 7, 1}}}},
     {{15},                        {{{2, 1}, {1, 3}}}},
     {{15, 12},                    {{{2, 1}, {1, 3}}, {{2, 2}, {2, 3, 10, 11}}}},
     {{15, 12, 15},                {{{2, 1}, {1, 3}}, {{2, 2}, {2, 3, 10, 11}}, {{2, 3}, {2, 3, 1, 8, 10, 11}}}},
```

Comment:
How about design with non-repeatable shapes. For example, {3, 4, 5, 6, 7} instead of {10, 9, 10, 9, 10}